server/src/services/uploadImage.js

```diff
@@ -0,0 +1,25 @@
+import AWS from "aws-sdk";
+import multer from "multer";
+import multerS3 from "multer-s3";
+import config from "../config.js";
+
+AWS.config.update({
+    accessKeyId: config.awsAccess.key,
+    secretAccessKey: config.awsSecret.key,
+    region: "us-east-1",
+});
+
+const s3 = new AWS.S3();
+
+const uploadImage = multer({
+    storage: multerS3({
+        s3: s3,
+        bucket: config.s3Bucket.name,
+        acl: "public-read",
+        key: function (req, file, cb) {
+            cb(null, /*file.originalname*/ Date.now().toString());
```

Comment:
we'll want to remove or use the code comment here for the file name

Reply:
As far as I know we did not add this code. Is it still safe to remove?

Reply:
This was me when testing this method (unsure which one was the best to use) and then I forgot to remove! thank you both